src/main/kotlin/io/spotfire/solver/plugins/Serialization.kt

```diff
@@ -0,0 +1,18 @@
+package io.spotfire.solver.plugins
+
+import io.ktor.serialization.kotlinx.json.*
+import io.ktor.server.application.*
+import io.ktor.server.plugins.contentnegotiation.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+
+fun Application.configureSerialization() {
+  install(ContentNegotiation) {
+    json()
+  }
+  routing {
+    get("/json/kotlinx-serialization") {
+      call.respond(mapOf("hello" to "world"))
+    }
+  }
+}
```

Comment:
should we add newlines at the end of these new files? not sure if that's an editorconfig or ide thing.

Reply:
yes, my mistake i configured the editor to add those and fixed the ones that had the weird missing linebreaks